includes/class-sensei-course.php

```diff
@@ -2655,7 +2657,7 @@ public static function get_loop_number_of_columns() {
 	public static function course_archive_sorting( $query ) {
 
 		// don't show on category pages and other pages
-		if ( ! is_archive( 'course ' ) || is_tax( 'course-category' ) ) {
+		if ( ! is_post_type_archive( 'course' ) || is_tax( 'course-category' ) ) {
```

Comment:
I fixed this (`is_archive` does not expect any parameter). And I think that in fact the second part (`|| is_tax( 'course-category` )` is not needed anymore either. But wanted to confirm before removing it.

Reply:
Hi @donnapep thanks! Maybe I am missing something but shouldn't the first part `! is_post_type_archive( 'course' )` be enough? I mean "anything that is not the archive page for courses" includes the category page too (at least in my tests).